web/src/testgrid-tab-table.ts

```diff
@@ -0,0 +1,61 @@
+import { LitElement, html, css } from 'lit';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { customElement, property } from 'lit/decorators.js';
+import {TabSummaryInfo} from './testgrid-dashboard-summary';
+
+@customElement('testgrid-tab-table')
+export class TestgridTabTable extends LitElement {
+  @property() clicked = '';
+  @property() visible = false;
+  info?: TabSummaryInfo;
```

Comment:
For this particular element we don't need the full tabSummaryInfo, right? We can just provide failuresSummary

Reply:
Yup, that's right, but actually right now I imported the `TabSummaryInfo`, but there's no data in this object when it gets called. So I'm still looking into this and trying to figure out why it's working in tab-summary but not this

Reply:
Done